test/integration/transpose_song.test.ts

```diff
@@ -96,4 +101,19 @@ Let it [Bbm]be, let it [Db/Ab]be, let it [Gb2]be, let it [Db]be`.substring(1);
     expect(updatedSong.key).toEqual('Db');
     expect(new ChordProFormatter().format(updatedSong)).toEqual(changedSheet);
   });
+
+  it('may be transposed without a key', () => {
+    const chordpro = `
```

Comment:
Just realized this variable should not be named `chordpro` since it uses the regular parser.